packages/types/src/sign-client/proposal.ts

```diff
@@ -17,7 +17,8 @@ export declare namespace ProposalTypes {
 
   export interface Struct {
     id: number;
-    expiry: number;
+    expiry?: number; // deprecated in favour of expiryTimespamp
```

Comment:
```suggestion
    expiry?: number; // deprecated in favour of expiryTimestamp
```

Reply:
Might be more useful to mark this as deprecated in a way TS/editors will pick up:
https://stackoverflow.com/questions/60755711/is-it-possible-to-mark-something-as-deprecated-in-typescript